cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -91,6 +91,38 @@ rules:
       - update
       - delete
 ---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
same changes are required in helm charts?

Reply:
Yes, and also OLM I'm looking into this.